modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -161,6 +162,35 @@
     /** */
     private static final String CMD_SKIP_ZEROS = "--skipZeros";
 
+    // SSL configuration section
+
+    /** */
+    protected static final String CMD_SSL_ENABLED = "--ssl_enabled";
+
+    /** */
+    protected static final String CMD_SSL_PROTOCOL = "--ssl_protocol";
+
+    /** */
+    protected static final String CMD_SSL_ALGORITHM = "--ssl_algorithm";
```

Comment:
Other parameters use "-" as words delimiter which is preferred to "_". It's also more UNIX convenient.